src/stores/queueStore.ts

```
@@ -351,42 +367,26 @@ export class TaskItemImpl {
-            return TaskItemDisplayStatus.Completed
-          case 'error':
-            return TaskItemDisplayStatus.Failed
-        }
+      case 'completed':
+        return TaskItemDisplayStatus.Completed
+      case 'failed':
+        return TaskItemDisplayStatus.Failed
+      case 'cancelled':
+        return TaskItemDisplayStatus.Cancelled
     }
   }
```

Biome (2.1.2)

[error] This getter should return a value.



(lint/suspicious/useGetterReturn)